utils/m0_clear_dump.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Personally, I think this can be a utility script to delete dump files/trace files/core files in a test. This is useful when doing a specific test case.

But to add it into the source code, I am not sure about the necessity.